apps/bfd-model/pom.xml

```diff
@@ -52,6 +39,12 @@
 				<artifactId>poi</artifactId>
 				<version>${poi.version}</version>
 			</dependency>
+			<dependency>
+				<!-- Makes programmatically generating Java sources much simpler. -->
+				<groupId>com.squareup</groupId>
+				<artifactId>javapoet</artifactId>
```

Comment:
Wouldnt build without it

Reply:
Likely was a transient dependency before; it had a version declared in here so it must have been being used.

Reply:
Gotcha resolving